packages/commonwealth/client/scripts/views/pages/Community/Members/CommunityMembersPage.tsx

```diff
@@ -138,7 +139,8 @@ const CommunityMembersPage = () => {
           : !(group.members || []).find(
               (x) => x?.address?.address === app.user.activeAccount.address
             )
-      );
+      )
+      .sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
since .sort will sort in-place/reference-update -- this will effect the react query cache, so I suggest we clone the `members` above i.e `[...members.pages]`

Reply:
@mzparacha @rbennettcw -- i know i have two approvals but can y'all re-check this with my newest commit https://github.com/hicommonwealth/commonwealth/pull/5512/commits/211a98af57d2bc4e0476af82b72edb11c87a32a8

Reply:
Resolving, thanks